client/state/posts/selectors.js

```diff
@@ -154,6 +195,13 @@ export function isRequestingSitePostsForQuery( state, siteId, query ) {
  * @return {?Number}        Total number of found items
  */
 export function getSitePostsFoundForQuery( state, siteId, query ) {
+	if ( ! siteId ) {
+		// TODO remove me after testing and before merge
+		// eslint-disable-next-line no-console
+		console.warn( 'getSitePostsFoundForQuery called without siteId' );
+		return null;
+	}
+
```

Comment:
This will probably need to be changed to handle all-sites queries and renamed to `getPostsFoundForQuery`.  See https://github.com/Automattic/wp-calypso/pull/19025/files#r147069571.